doc/user/layouts/shortcodes/self-managed/self-managed-details.html

```diff
@@ -6,3 +6,5 @@
 | Maximum Usage Limits (Memory) | None | 24 GiB |
 | Maximum Usage Limits (Disk) |  None | 48 GiB |
 | Support | Per terms of your license | Community slack |
+
+See also [Self-Managed Community Edition Privacy FAQ](/privacy-faq/).
```

Comment:
so, this will appear
https://preview.materialize.com/materialize/31799/self-managed/v25.1/
https://preview.materialize.com/materialize/31799/self-managed/v25.1/get-started/#self-managed-materialize